backend/go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	golang.org/x/net v0.0.0-20191116160921-f9c825593386 // indirect
 	golang.org/x/sys v0.0.0-20191118133127-cf1e2d577169 // indirect
+	golang.org/x/tools v0.0.0-20191203043002-5ae4576c3a41 // indirect
```

Comment:
I wonder why is this added?

Reply:
these are auto generated

Reply:
@dawntomm May I ask what command did you use to automatically generate them?